docs/DafnyRef/UserGuide.md

```diff
@@ -371,8 +371,8 @@ For the formatter to work, the file should be parsed correctly by Dafny.
 
 There are four ways to use the formatter:
 
-* `dafny format file1.dfy file2.dfy ...` formats each file provided in arguments.
-* `dafny format --print:- file1.dfy file2.dfy ...` format each file but instead of writing files, output their formatted content in stdout
+* `dafny format file1.dfy file2.dfy ...` formats each file provided in arguments, altering the files in place.
+* `dafny format --print file1.dfy file2.dfy ...` format each file but instead of altering the files, output the formatted content to stdout
 * `dafny format` Look up for all the Dafny files recursively in the current repository and format them.
```

Comment:
But I still want to see the community's take on this behavior -- I really want there to be some argument required for there to be such a big change.